modules/apps/ccv/utils/utils.go

```diff
@@ -0,0 +1,24 @@
+package utils
+
+import (
+	abci "github.com/tendermint/tendermint/abci/types"
+)
+
+func AccumulateChanges(currentChanges, newChanges []abci.ValidatorUpdate) []abci.ValidatorUpdate {
```

Comment:
Maybe this could go in the types package? I think even just a file on the top level could be fine. I'd recommend avoiding `utils/utils.go` and go with file naming that is descriptive

Function needs godoc/comments/tests as well